addons/website_project_issue/models/project_issue.py

```diff
@@ -1,13 +1,29 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, exceptions, models, _
+from odoo import api, fields, exceptions, models
 
 
 class Issue(models.Model):
     _name = "project.issue"
     _inherit = ['project.issue']
 
+    main_attachment_ids = fields.One2many('ir.attachment', compute='_compute_attachment_ids', string="Main Attachments",
+                                          help="Attachment that don't come from message.")
+    website_url = fields.Char('Website URL', compute='_compute_website_url', help='The full URL to access the document through the website.')
+
+    def _compute_website_url(self):
+        for issue in self:
+            issue.website_url = '/my/issues/%s' % issue.id
+
+    def _compute_attachment_ids(self):
```

Comment:
Yes, it will be so much simple if you don't save attachment from message on the 'main record' but on the message it self !
I ask you  more than one month ago how to get it, you just say "tde@odoo.com has left the conversation." ! Thanks for your opinion !